src/main/java/org/bitnick/portfolio/service/ProjectService.java

```diff
@@ -46,6 +87,13 @@ public Project saveProject(Project project) {
         }
     }
 
+    /**
+     * Updates a Project to the database. Any exception thrown will be caught,
+     * logged, and a User object will be returned will "null" properties.
+     * @param project Project entity with modified properties to be updated
+     * @param id UUID of target Project to be updated
+     * @return Project, updated Project entity
+     */
```

Comment:
Redundant to the other service javadocs, however this is closer to the desired docs